modules/proc-switching-bf2-nic.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * networking/switching-bf2-nic-dpu.adoc
+
+:_content-type: PROCEDURE
+[id="proc-switching-bf2-nic_{context}"]
+= Switching Bluefield-2 from DPU mode to NIC mode
+
+Use the following procedure to switch Bluefield-2 from data processing units (DPU) mode to network interface controller (NIC) mode.  
```

Comment:
I think Bluefield-2 is a type of thing, a network device?

Reply:
yes